<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Coordinates to set equal for both objects</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Second Object</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>First Object</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="yCheck">
     <property name="text">
      <string>y axis</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="xrotCheck">
     <property name="text">
      <string>x axis rotation</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="yrotCheck">
     <property name="text">
      <string>y axis rotation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="secondObjectList"/>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="firstObjectList"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="zCheck">
     <property name="text">
      <string>z axis</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="xCheck">
     <property name="text">
      <string>x axis</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="zrotCheck">
     <property name="text">
      <string>z axis rotation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
